### DISCUSSION TITLE
Geo_preview issue with CKan 2.10

### DISCUSSION THREAD

**Discussion by tawbe** [ckan/ckan#7910] [Q&A]
Hello everyone, can someone help me? I installed Ckan 2.10, I work with python3 and I installed geo-preview when I add it to the list of plugins I got this error : raise AttributeError(f"module '{name}' has no attribute '{name}'")
AttributeError: module 'ckan.plugins' has no attribute 'IRoutes'

**Comment by pdelboca** [comment]
`IRoutes` has been deprecated. So make sure no plugin implements it.

Also `ckan.plugins` should not be a plugin itself, check your configuration file is well formated.
